_episodes/l2-04-colaborative-development.md

```diff
@@ -34,37 +34,53 @@ knowledge acquired so far.
+>
+>   ![Use this template]({{ site.baseurl }}/fig/use_this_template.png "Use this template"){:class="img-responsive" style="float: left; margin-right: 10px; width: 20%;"}
+>
+>   <div style="clear: both;"></div>
+>
+> - Set the owner of the new repository as your own GitHub username and the name of the repository the same ("book_of_recipes").
+> - Set the description as "Repository for the exercises of the Grad School Git Course".
+> - Make sure to select the "Public" option for your reppsitory's visibility and final click on "Create repository".
+> - Pass the link to the repo to your colleagues, who will be the contributors.
 >
 > Now, start collaborating!
 >
-> - The administrator creates a new release, let's say `1.0.0`, as the starting point of
->   the book.
-> - All, including the administrator, open new issues with recipes for sauces you
->  will like to have in the book.
-> - Administrator, add some tags, prioritising some of the recipes, and assign
->  yourself or one of your colleagues as responsible for each of them.
->  Remember that if you do not have write access to the repository (which is the case here)
->  and you are not part of the same organisation of the repository, the only way of being
->  assigned to an Issue is by making a comment on the Issue.
-> - Fork the administrator's repository. Administrator, did you notice how the
->  number of forks increases? Which GitHub users forked it from you? And from
->  the original repo?
-> - Work on the recipes you have been assigned. Practice the concepts learnt
->  in previous episodes about cloning a repository, making the changes locally
->  and pushing those changes back to the remote repository. You can even try a
->  [gitflow approach](https://nvie.com/posts/a-successful-git-branching-model/)
->  if you feel ambitious!
-> - When ready, open a PR to the administrator's repo and request their review.
-> - Administrator, review the PR, request some changes and accept others. Make sure the
->   relevant checks performed by the continuous integration system are all passing.
+> **Create a new release** (Administrator's task)
+>
+> - Create a new release, let's say `1.0.0`, as the starting point of the book.
+>
+> **Creating new issues** (to be done by everyone):
+>
+> - Open new issues with recipes for sauces you would like to have in the book.
+>
+> **Explore existing issues** (Contributor's task):
+>
+> - Comment on the issues that you want to be assigned to. (As you don't have write access to the repository, you cannot assign yourself to an issue directly.)
+>
+> **Task assignment** (Administrator's task):
+>
+> - Add some tags, prioritise some of the recipes, and assign yourself or one of your colleagues as responsible for each of them.
+>
+> **Working on a branch, pushing the changes, and opening a PR** (Contributor's tasks):
+>
+> - Fork the Administrator's repository and clone it locally.
```

Comment:
I'd be inclined to but this forking task at the top, right after the Housekeeping section. Just so they can get set up while the administrator creates the release